.gitignore

```diff
@@ -24,3 +24,5 @@ htmlcov/
 .vscode/
 *.swp
 *.swo
+
+.env
```

Comment:
✅ Adding '.env' to .gitignore is a good practice to prevent sensitive information from being tracked!